README.md

```diff
@@ -1,2 +1,90 @@
-# generic-translator
-Module for translating commands in device compatible set based on manufacturer and device type
+# Generic Translator Module
+
+|                |                                     |
+| -------------- | ----------------------------------- |
+| Name           | Generic Translator Module           |
+| Version        | v1.0.0                              |
+| Dockerhub Link | [weevenetwork/generic-translator]() |
+| Authors        | Mesud Pasic                         |
+
+- [MQTT Ingress](#generic-translator)
+  - [Description](#description)
+  - [Features](#features)
+  - [Environment Variables](#environment-variables)
+    - [Module Specific](#module-specific)
+    - [Set by the weeve Agent on the edge-node](#set-by-the-weeve-agent-on-the-edge-node)
+  - [Dependencies](#dependencies)
+
+## Description
+
+Encoder for Vicki LoRaWAN's commands.
```

Comment:
Please be more informative what this module is doing.

Reply:
set it same as in Module.yaml, for the same reasons I mentioned above.